fme.md

```diff
@@ -4,495 +4,532 @@ N/A means "not applicable" in this context
+[2DBoxReplacer](https://www.safe.com/transformers/2d-box-replacer/)|N/A|Needs expression functions to create circular strings
+[2DEllipseReplacer](https://www.safe.com/transformers/2d-ellipse-replacer/)|N/A|Needs expression functions to create circular strings
+[2DForcer](https://www.safe.com/transformers/2d-forcer/)|Drop Z Value|Missing
+[2DGridAccumulator](https://www.safe.com/transformers/2d-grid-accumulator/)|Create grid (with extend option)|N/A
+[2DGridCreator](https://www.safe.com/transformers/2d-grid-creator/)|Create grid|N/A
+[3DAffiner](https://www.safe.com/transformers/3d-affiner/)|Affine transform|N/A
+[3DArcReplacer](https://www.safe.com/transformers/3d-arc-replacer/)|Missing|N/A
+[3DForcer](https://www.safe.com/transformers/3d-forcer/)|Set Z Value|Missing
+[3DInterpolator](https://www.safe.com/transformers/3d-interpolator/)|Missing|N/A|Also desirable for m values|
+[3DRotator](https://www.safe.com/transformers/3d-rotator/)|Rotate|N/A
+[Affiner](https://www.safe.com/transformers/affiner/)|Affine transform|Missing
+[AffineWarper](https://www.safe.com/transformers/affine-warper/)|Missing|N/A|Would use gdal to perform warp via gcps -- could expose more than just affine transformation. Possibly aided by Lutra's upcoming work improving the QGIS georeferencer.|
+[AggregateFilter](https://www.safe.com/transformers/aggregate-filter/)|Missing|N/A
+[Aggregator](https://www.safe.com/transformers/aggregator/) |Aggregate, Collect geometries|collect
+[AmazonAthenaConnector](https://www.safe.com/transformers/amazon-athena-connector/)|Missing|N/A
+[AnchoredSnapper](https://www.safe.com/transformers/anchored-snapper/)|Snap geometries to layer|Missing|Performance issues |
+[AngleConverter](https://www.safe.com/transformers/angle-converter/)|Missing|Needs expression functions to create circular strings
+[AngularityCalculator](https://www.safe.com/transformers/angularity-calculator/)|Missing|Needs expression functions to create circular strings
+[AppearanceExtractor](https://www.safe.com/transformers/appearance-extractor/) |Missing|Needs expression functions to create circular strings|Doesn't apply in QGIS|
+[AppearanceMerger](https://www.safe.com/transformers/appearance-merger/)|Missing|Needs expression functions to create circular strings|Doesn't apply in QGIS|
+[AppearanceRemover](https://www.safe.com/transformers/appearance-remover/)|Missing|Needs expression functions to create circular strings|Doesn't apply in QGIS|
+[AppearanceSetter](https://www.safe.com/transformers/appearance-setter/)|Missing|Needs expression functions to create circular strings|Doesn't apply in QGIS|
+[AppearanceStyler](https://www.safe.com/transformers/appearance-styler/)|Missing|Needs expression functions to create circular strings|Doesn't apply in QGIS|
+[ArcEstimator](https://www.safe.com/transformers/arc-estimator/)|Missing|N/A|Port from PostGIS? Partial work at https://github.com/nyalldawson/QGIS/tree/convert_to_curves|
```

Comment:
I think this was completed by someone else a couple of years ago

Reply:
Should we remove this reference then?